app.js

```diff
@@ -79,7 +79,7 @@ app.use(session({
 	resave: false,
 	saveUninitialized: false
 }));
-
+app.use(compression());
```

Comment:
would be better to do it on proxy if we have one https://github.com/goldbergyoni/nodebestpractices/blob/master/sections/production/delegatetoproxy.md